scripts/constants.ts

```diff
@@ -57,3 +57,6 @@ export const threePoolAmplificationUtilsAddress = "0x114ECaa70256aFAd393f733aA4B
 export const threePoolSwapUtilsAddress = "0x0564d68404608599e8c567A0bD74F90a942A69A0";
 export const threePoolSwapFlashLoanAddress = "0x458459E48dbAC0C8Ca83F8D0b7b29FEfE60c3970";
 export const threePoolLpTokenAddress = "0x87BCC091d0A7F9352728100268Ac8D25729113bB";
+
+// StableTriMaker
+export const stableTriMaker = "0x3828bEa669d6aBc589bf07b4CA8fe6F427F27648";
```

Comment:
@rej156 i think we should deploy another permanent one to prod, this one is just for testing

Reply:
yeah, lets not merge this till we have a final `StableTriMaker` deployed

Reply:
+1 dw